src/handlers/componentInstall.ts

```diff
@@ -135,3 +146,26 @@ export default async function componentInstall(
     }
   }
 }
+
+function buildComponentDependencyList(components: Components, name: string) {
```

Comment:
Though this technically works, I believe this function should live in the `/src/util` directory in a separate file so we can add tests around it and make use of it elsewhere in the system as needed. This would be similar to how we implement `installComponentFromCache` in this file as well.

Reply:
What is the best path for this function? src/util/project? I am going to add test later this weekend.

Reply:
Yes I think that is probably the best spot for it for now. src/util/project

Reply:
Function extracted as a separate element and covered with unit tests. All functionality tested.